public/scripts/init.js

```diff
@@ -51,4 +51,75 @@ document.addEventListener("account.create", function () {
   }, function (error) {
     window.location = '/auth/signup?failure=1';
   });
-});
\ No newline at end of file
+});
+window.addEventListener("load", async () => {
```

Comment:
I think this part of the code can definitely be part of the new chart component. Beside that, all looks good.

Reply:
Definitely 👍🏻